windpowerlib/power_output.py

```diff
@@ -238,14 +228,26 @@ def power_curve_density_correction(
             Variable Scale Simulation Model for Windpower based on the
             Georeferenced Installation Register of Germany". Master's Thesis
             at Reiner Lemoine Institute, 2014, p. 13
-
     """
     if density is None:
         raise TypeError(
             "`density` is None. For the calculation with a "
             + "density corrected power curve density at hub "
             + "height is needed."
         )
+
+    # NOTE : CHANGES ARE MADE HERE FOR SPEED IMPROVEMENT
+    # create a flag for pandas Series type
+    Panda_series = False
```

Comment:
1. In the end it is static code but your comment is just for now. I would prefer something like: "Convert pandas.Series to a numpy array to speed up the interpolation below." (just a suggestion)
2. For me it would be clearer like this: `if....: pandas_series = True .. else: pandas_series = False`
3. `pandas_series` is a variable and therefore lower case in Python.